### ISSUE TITLE
After new person is added or a person is edited and save is clicked, redirect to the people page.

### ISSUE THREAD

**Issue by aliciabeckett-gov** [CDCgov/prime-central#169]
**Describe your feature request in this format: As a __(persona)___ I want to ___(action or feature)__ so that ___(expected outcome)__**
A clear and concise description of what the problem, user persona, and expected outcome.

**Describe the solution you'd like and the specific acceptance criteria to be completed**

**Describe alternatives you've considered**

**Additional context**

**Comment by TomNUSDS** [comment]
This is a duplicate of this bug from 4 days ago in prime-data-client. https://github.com/CDCgov/prime-data-input-client/issues/196 I'll close that one and make this the primary.

**Comment by aliciabeckett-gov** [comment]
Looking at this on the demo environment and I think there is a bug.

After the page redirects, the people page is blank. You have to refresh to see the people. @TomNUSDS

**Comment by TomNUSDS** [comment]
@timothybest-usds Yeah, this was caused by an interaction with Tim's change last night. The pre-coffee change https://github.com/CDCgov/prime-data-input-client/pull/255 fixes it.
